objc/core/A0LockReact.m

```diff
@@ -140,16 +141,38 @@ - (void)showWithOptions:(NSDictionary *)options callback:(A0LockCallback)callbac
     self.shown = YES;
 }
 
+- (void)getDelegationToken:(NSDictionary *)options callback:(A0LockCallback)callback {
+    if (!self.lock) {
+        callback(@[@"Please configure Lock before using it"]);
+        return;
+    }
+    A0APIClient *client = [self.lock apiClient];
+
+    void(^success)(NSDictionary *) = ^(NSDictionary *credentials) {
+        callback(@[[NSNull null], credentials]);
+    };
+    void(^failure)(NSError *) = ^(NSError *error) {
+        callback(@[[error localizedDescription]]);
+    };
+
+    A0AuthParameters *parameters = [A0AuthParameters newWithDictionary:@{
```

Comment:
looking at the rest of codebase there is no check for auth params which also makes sense because the methods would crash the app if some forgets to pass it. They all are mandatory so crash makes sense. Am i missing anything?